ansible/README.rst

```diff
@@ -73,6 +85,25 @@ The remaining of this guide uses OVN-SCALE-TEST as the image name. You need to
 change the name in your deployment.
 
 
+Build upstream OVS/OVN images
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The pre-built images stored on Dockerhub may not include the latest OVS commit.
+To test latest OVS/OVN, the following steps need to be run
+
+1. Edit ``ansible/docker/ovn/Makefile`` and run ``make`` to rebuild the image. In ``ansible/docker/ovn/Makefile``, the variable ``image_name`` should be set appropriately, e.g.,
```

Comment:
I'm sorry to keep nit-picking, but I really think these should be line wrapped at 80 characters. :)